.ci/pytorch/test.sh

```diff
@@ -1641,7 +1641,7 @@ elif [[ "${TEST_CONFIG}" == *torchbench* ]]; then
     install_torchaudio cuda
   fi
   install_torchvision
-  TORCH_CUDA_ARCH_LIST="8.0;8.6" pip_install git+https://github.com/pytorch/ao.git
+  TORCH_CUDA_ARCH_LIST="8.0;8.6" install_torchao
```

Comment:
Let me try to see if this works

```suggestion
  TORCH_CUDA_ARCH_LIST="8.0" install_torchao
```